### PR TITLE
update hammer_cli(_foreman) to 0.2.0 (DEB)

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

File: dependencies/precise/hammer_cli/control [hunk 1]
```diff
@@ -21,7 +21,7 @@ Depends: ${shlibs:Depends}, ${misc:Depends}, ruby | ruby-interpreter,
          ruby-fastercsv,
          ruby-mime-types (< 2.0.0),
          ruby-fast-gettext (>= 0.8.0),
-         liblocale-ruby1.8 (<= 2.0.9) | liblocale-ruby1.9.1 (<= 2.0.9),
+         liblocale-ruby1.8 (>= 2.0.6) | liblocale-ruby1.9.1 (>= 2.0.6),
```
Comment (domcleal) [inline dependencies/precise/hammer_cli/control]: 12.04 seems to have 2.0.5: http://packages.ubuntu.com/search?keywords=liblocale-ruby

File: dependencies/wheezy/hammer_cli/control [hunk 2]
```diff
@@ -20,7 +20,7 @@ Depends: ${shlibs:Depends}, ${misc:Depends}, ruby | ruby-interpreter,
          ruby-fastercsv,
          ruby-mime-types (< 2.0.0),
          ruby-fast-gettext (>= 0.8.0),
-         ruby-locale (<= 2.0.9),
+         ruby-locale (>= 2.0.6),
```
Comment (domcleal) [inline dependencies/wheezy/hammer_cli/control]: https://packages.debian.org/wheezy/ruby-locale also

**Comment by mmoll** [comment]
Right, added locale 2.0.8 packages for wheezy and precise.

**Comment by domcleal** [comment]
Thanks @mmoll!  Please go ahead and build.

**Pull request merged**